conda_forge_tick/migrators.xsh

```diff
@@ -151,6 +159,47 @@ class Migrator:
             node for node in graph if not list(graph.predecessors(node)))
         return cyclic_topological_sort(graph, top_level)
 
+    @classmethod
+    def set_build_number(cls, filename):
+        """Bump the build number of the specified recipe.
+
+        Parameters
+        ----------
+        filename : str
+            Path the the meta.yaml
+
+        """
+
+        for p, n in cls.build_patterns:
+            with open(filename, 'r') as f:
+                raw = f.read()
+            lines = raw.splitlines()
+            for i, line in enumerate(lines):
+                m = p.match(line)
+                if m is not None:
+                    old_build_number = int(m.group(2))
+                    new_build_number = cls.new_build_number(old_build_number)
+                    lines[i] = m.group(1) + n.format(new_build_number)
+            upd = '\n'.join(lines) + '\n'
+            with open(filename, 'w') as f:
+                f.write(upd)
+
+    @classmethod
+    def new_build_number(cls, old_number: int):
```

Comment:
@CJ-Wright this is the default behavior

Reply:
Ah ok, sorry for the confusion

Reply:
It is OK, I understand you haven't been coding for a while :smirk: